### ISSUE TITLE
Toggling semantics debugger leads to crash

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#13957]
Steps to reproduce:
1. Launch Stocks
2. Open drawer
3. Tap Settings
4. Tap "Show semantics overlay" three times (on, off, on)

```
I/flutter (20775): ══╡ EXCEPTION CAUGHT BY SCHEDULER LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter (20775): The following assertion was thrown during a scheduler callback:
I/flutter (20775): 'package:flutter/src/semantics/semantics.dart': Failed assertion: line 797 pos 12:
I/flutter (20775): 'owner._nodes.containsKey(id)': is not true.
I/flutter (20775): 
I/flutter (20775): Either the assertion indicates an error in the framework itself, or we should provide substantially
I/flutter (20775): more information in this error message to help you determine and fix the underlying cause.
I/flutter (20775): In either case, please report this assertion by filing a bug on GitHub:
I/flutter (20775):   https://github.com/flutter/flutter/issues/new
I/flutter (20775): 
I/flutter (20775): When the exception was thrown, this was the stack:
I/flutter (20775): #2      SemanticsNode.detach (package:flutter/src/semantics/semantics.dart:797:12)
I/flutter (20775): #3      AbstractNode.dropChild (package:flutter/src/foundation/node.dart:143:13)
I/flutter (20775): #4      SemanticsNode._replaceChildren (package:flutter/src/semantics/semantics.dart:706:27)
I/flutter (20775): #5      SemanticsNode.updateWith (package:flutter/src/semantics/semantics.dart:935:5)
I/flutter (20775): #6      RenderSemanticsGestureHandler.assembleSemanticsNode (package:flutter/src/rendering/proxy_box.dart:2748:10)
I/flutter (20775): #7      _SwitchableSemanticsFragment.compileChildren (package:flutter/src/rendering/object.dart:3190:13)
I/flutter (20775): #8      _SyncIterator.moveNext (dart:core-patch/dart:core/core_patch.dart:149)
I/flutter (20775): #9      List.addAll (dart:core-patch/dart:core/growable_array.dart:188)
I/flutter (20775): #10     _RootSemanticsFragment.compileChildren (package:flutter/src/rendering/object.dart:3094:16)
I/flutter (20775): #11     _SyncIterator.moveNext (dart:core-patch/dart:core/core_patch.dart:149)
I/flutter (20775): #12     Iterable.single (dart:core/iterable.dart:512)
I/flutter (20775): #13     RenderObject._updateSemantics (package:flutter/src/rendering/object.dart:2275:96)
I/flutter (20775): #14     PipelineOwner.flushSemantics (package:flutter/src/rendering/object.dart:872:16)
I/flutter (20775): #15     BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:274:19)
I/flutter (20775): #16     BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:613:22)
I/flutter (20775): #17     BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:208:5)
I/flutter (20775): #18     BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:921:15)
I/flutter (20775): #19     BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:861:9)
I/flutter (20775): #20     BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:773:5)
I/flutter (20775): #21     _invoke (file:///b/build/slave/Linux_Engine/build/src/flutter/lib/ui/hooks.dart:120)
I/flutter (20775): #22     _drawFrame (file:///b/build/slave/Linux_Engine/build/src/flutter/lib/ui/hooks.dart:109)
I/flutter (20775): (elided 2 frames from class _AssertionError)
I/flutter (20775): ════════════════════════════════════════════════════════════════════════════════════════════════════
```

cc @goderbauer

**Comment by goderbauer** [comment]
@Hixie I am unable to reproduce this on master on my Nexus 6P. Does this consistently reproduce for you? Do you have any additional information about your environment that might be relevant?

**Comment by goderbauer** [comment]
Ah, never mind. I had TalkBack running in the background (with feedback paused). When I turn off talkback completely in the settings the issue reproduces as described. Will take a look.
